crates/eframe/Cargo.toml

```diff
@@ -121,7 +121,7 @@ image = { version = "0.24", default-features = false, features = [
   "png",
 ] } # Needed for app icon
 raw-window-handle = { version = "0.5.0" }
-winit = { version = "0.28.1", default-features = false }
+winit = { version = "0.29.2", default-features = false, features = ["rwh_05"] }
```

Comment:
In [`raw-window-metal`](https://github.com/norse-rs/raw-window-metal/issues/11) and [`ash-window`](https://github.com/ash-rs/ash/pull/799) we're also debating how to best support the new feature-based (== additive!) compatibility with multiple `raw-window-handle` versions in **consumer crates**. This seems easiest with a trait (as demonstrated in **handle producer** crates like `winit` and `ndk`), but especially in `ash-window` would also leads to lots of `match` duplication (unless we convert from one handle to another).

Reply:
Personally I feel like we should push to just get everyone on board with the latest `raw-window-handle` ASAP instead of maintaining compatibility with different versions using features across a lot of crates. But don't know how realistic that is :)?

Reply:
It is a fair point for consumer crates, though things in-between like `ash-window` and `raw-window-metal` will have a hard time catering to both sides. `eframe` likely feels like an in-between crate.

In the case of `ash-window` I even have to consider _when_ to release the crate if I don't embed backwards compatibility, as there's a direct link to its parent `ash` crate which is "on the brink of" a semver-breaking release. I either give people a new version of `ash-window` with compatibility for `raw-window-handle 0.6` - and then force the next release of `ash 0.38` to only be compatible with `raw-window-handle 0.6` - or release `ash 0.38` first together with a matching breaking release of `ash-window` that is still on `raw-window-handle 0.5`, not giving folks on `ash 0.37` a way to upgrade (and they'll have quite some breaking changes to take care of in their code, even if trivially find-replaceable).